### PR TITLE
RA-429 : add liquibase to delete the duplicate concepts

### PR DESCRIPTION
**Pull request by mozzy11**
https://issues.openmrs.org/browse/RA-429

### CODE REVIEW

File: api/pom.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
 	<parent>
 		<groupId>org.openmrs.module</groupId>
 		<artifactId>referencemetadata</artifactId>
-		<version>2.9.0-SNAPSHOT</version>
+		<version>2.10.0-SNAPSHOT</version>
```
Comment (dkayiwa) [inline api/pom.xml]: Is this change part of the ticket?

File: omod/pom.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
 	<parent>
 		<groupId>org.openmrs.module</groupId>
 		<artifactId>referencemetadata</artifactId>
-		<version>2.9.0-SNAPSHOT</version>
+		<version>2.10.0-SNAPSHOT</version>
```
Comment (dkayiwa) [inline omod/pom.xml]: Is this change part of the ticket?

File: pom.xml [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 	<groupId>org.openmrs.module</groupId>
 	<artifactId>referencemetadata</artifactId>
-	<version>2.9.0-SNAPSHOT</version>
+	<version>2.10.0-SNAPSHOT</version>
```
Comment (dkayiwa) [inline pom.xml]: Is this change part of the ticket?

File: api/pom.xml (hunk 1 above)
Comment (mozzy11) [reply api/pom.xml]: oop s , that was a mistake , thats unnecesary. let me clean it out

**Comment by mozzy11** [comment]
i removed the unnecessary commits

**Pull request closed without merging**